plugin/pkg/shoot/validator/admission_test.go

```diff
@@ -2528,6 +2529,30 @@ var _ = Describe("validator", func() {
 
 			})
 
+			It("should not reject because shoot services network is nil for IPv6 (workerless Shoot)", func() {
```

Comment:
Add another test, which doesn't set the the network ranges to random value, instead set it to shoot defaults from seed, if shoot default specifies IPv6 network ranges.